firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -53,6 +54,11 @@ expect class Transaction {
 //    val persistenceEnabled: Boolean = true
 //)
 
+enum class QueryDirection {
```

Comment:
Please check if it's still needed, because in orderBy used another Direction type

Reply:
7b8f448